[2023-04-25T13:22:48.449+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:22:47.100102+00:00 [queued]>
[2023-04-25T13:22:48.460+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:22:47.100102+00:00 [queued]>
[2023-04-25T13:22:48.460+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T13:22:48.465+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-25T13:22:48.465+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T13:22:48.487+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-25 13:22:47.100102+00:00
[2023-04-25T13:22:48.505+0000] {standard_task_runner.py:55} INFO - Started process 13545 to run task
[2023-04-25T13:22:48.523+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-25T13:22:47.100102+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp_2vi_9na']
[2023-04-25T13:22:48.526+0000] {standard_task_runner.py:83} INFO - Job 62: Subtask is_api_available
[2023-04-25T13:22:48.629+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:22:47.100102+00:00 [running]> on host bebb68623a17
[2023-04-25T13:22:48.770+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T13:22:47.100102+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-25T13:22:47.100102+00:00
[2023-04-25T13:22:48.771+0000] {http.py:120} INFO - Poking: api/
[2023-04-25T13:22:48.800+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-25T13:22:48.802+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-25T13:22:50.139+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-25T13:22:50.169+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230425T132247, start_date=20230425T132248, end_date=20230425T132250
[2023-04-25T13:22:50.208+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-25T13:22:50.229+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
